Documentation/ApiOverview/Events/Events/Backend/_AfterRawPageRowPreparedEvent/_MyEventListener.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Backend\EventListener;
+
+use TYPO3\CMS\Backend\Tree\Repository\AfterRawPageRowPreparedEvent;
+use TYPO3\CMS\Core\Attribute\AsEventListener;
+
+final class MyEventListener
+{
+    #[AsEventListener]
+    public function __invoke(AfterRawPageRowPreparedEvent $event): void
+    {
+        $rawPage = $event->getRawPage();
+        if ((int)$rawPage['uid'] === 123) {
+            // Sort pages alphabetically in the page tree
+            $rawPage['_children'] = usort($page['_children'], function (array $a, array $b) {
+                return strcmp($a['title'], $b['title']);
```

Comment:
Do you know Star Wars? Let's try spaceship here ;-)

Reply:
Thought also about it, but haven't looked up the details of `strcmp()` and if it differs from spaceship ;-)

Also, I prefer Star Trek :star:

Reply:
Okay, strcmp is binary-safe and compares only strings, spaceship can compare "any" type, maybe with type adjustment. Is this relevant in this case? I don't think so (also considering UTF-8). So, IMHO, go for it.

Reply:
I would keep strcmp for utf-8 here

Reply:
strcmp is binary safe, but as page title is just varchar and no blob, we don't need binary safety here. Further strcmp returns values greater than -1 and 1. But for just comparing, spaceship should be enough.

I like Chris Pine...do you'll still talk to me :D

Reply:
I haven't seen those films with him, so no opinion from me ;-)